contracts/hydro/src/query.rs

```diff
@@ -148,7 +157,13 @@ pub struct LockupWithPerTrancheInfo {
 
 #[cw_serde]
 pub struct AllUserLockupsResponse {
-    pub lockups: Vec<LockupWithPerTrancheInfo>,
+    pub lockups: Vec<LockEntryWithPower>,
+}
+
+// A version of AllUserLockupsResponse that includes the per-tranche information for each lockup.
+#[cw_serde]
+pub struct AllUserLockupsWithTrancheInfosResponse {
```

Comment:
This response type must be added to hydro_schema.rs to generate JSON schema and TS clients properly.

Reply:
thx for the diligence!